lms/static/js/certificates/models/certificate_exception.js

```diff
@@ -24,11 +24,9 @@
                     certificate_generated: '',
                     notes: ''
                 },
-
-               url: function() {
-                    return this.get('url');
+                initialize: function (attributes, options) {
```

Comment:
@aamir-khan we were passing as attr, but the backbone upgrade it is not working so we need to pass it in the options.